src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs

```diff
@@ -164,6 +166,8 @@ public async Task<FormulaValue> EvalAsync(string expressionText, CancellationTok
             var parameterSymbols = runtimeConfig?.Values?.SymbolTable;
             var symbolsAll = ReadOnlySymbolTable.Compose(parameterSymbols, symbolTable);
 
+            options ??= new ParserOptions() { ReservedKeywords = Config.Features.ReservedKeywords };

```

Comment:
this is the wrong place.  It hides nulls.   Can you update Engine.GetDefaultParserOptionsCopy()?

Reply:
Thanks, done!  GetDefaultParserOptionsCopy() has already been updated.